fastlane/Fastfile

```diff
@@ -135,11 +136,11 @@ platform :ios do
   #####################################################################################
   desc 'Updates the AppStoreStrings.pot file with the latest data'
   lane :update_appstore_strings do |options|
-    prj_folder = Pathname.new(File.join(Dir.pwd, '..')).expand_path.to_s
-    source_metadata_folder = File.join(prj_folder, 'fastlane/appstoreres/metadata/source')
+    project_root = File.dirname(File.expand_path(__dir__))
```

Comment:
I'm guessing you went for `File` instead of `Pathname` because we don't need to do any path operation here and this saves the awkward `.to_s` call?

Reply:
Yeah and also because tbh this changed twice while I was working on this:
 - When I started that branch a while ago, there was the `PROJECT_ROOT` top-level constant (around [those lines](https://github.com/woocommerce/woocommerce-ios/blob/218cb5ea100d79a98a031fc273b817e4f676c235/fastlane/Fastfile#L6-L8)), so at that time I just removed the `prj_folder` and then directly used `source_metadata_folder = File.join(PROJECT_ROOT, …)` instead of where `prj_folder` was used.
 - But then your PR to move secrets outside the project got rid of that `PROJECT_ROOT` top-level constant, so when I came back to fix the conflicts on this PR and rebased it, it didn't exist anymore. Instead of re-introducing the top-level constant, I decided to just define it as a local variable as before, just borrowing how it was previously defined as `PROJECT_ROOT`, but keeping it in the scope of this lane only because that's the only place it was used.

I didn't really think twice about the fact that those 2 steps ended up changing a call using `Pathname` into a call just using `File`, but otoh, since we indeed don't really need complex path operations here, I end up also feeling that using `File` methods is nice here anyway 😛 

---

To be honest I am not really sure we need to specify `project_root` in those paths (and provide the toolkit with full, absolute paths) or if the toolkit would work just as well if we just passed `fastlane/appstoreres/metadata/source` as a relative path, just like we pass `WooCommerce/Resources` to `ios_lint_localizations` below. Which, if that works, could mean we'd be able to totally get rid of the project_root variable (or constant) – which would be a nice step in the direction of https://github.com/wordpress-mobile/release-toolkit/issues/205 too btw. But I figured this would be a problem for another PR 😅

Reply:
>  But I figured this would be a problem for another PR 😅

Sounds good to me 😄 🙌